src/Components/Admin/DeleteUser.js

```diff
@@ -54,10 +60,11 @@ function DeleteUser({ close, userId, setShowSaveFailure, setIsSubmitting }) {
   return (
     <QuestionWrapper>
       <hr />
-      <StyledLabel>Delete user?</StyledLabel>
+      <StyledLabel>Remove user from this program?</StyledLabel>
       <WarningMessage>
         <p>
-          Careful, you will delete the user and all associated data forever.
+          If they have conducted any reviews, they will continue to show in
+          reports and the Committee Review page.
```

Comment:
We should ensure any review data still shows here! I've made [this low-priority ticket](https://www.notion.so/uwblueprintexecs/Active-Epics-95f65fd1b589465e9adbe5ead3815e2e?p=73245f6b3ec842ef892ed622bfb09b1c) to track this work.